pkg/service/credential/storage.go

```diff
@@ -225,7 +262,7 @@ func (b BoltCredentialStorage) getCredential(id string, namespace string) (*Stor
 // GetCredentialsByIssuer gets all credentials stored with a prefix key containing the issuer value
 // The method is greedy, meaning if multiple values are found and some fail during processing, we will
 // return only the successful values and log an error for the failures.
-func (b BoltCredentialStorage) GetCredentialsByIssuer(issuer string) ([]StoredCredential, error) {
+func (b *CredentialStorage) GetCredentialsByIssuer(issuer string) ([]StoredCredential, error) {
 	keys, err := b.db.ReadAllKeys(credentialNamespace)
```

Comment:
I really like this change but I'm curious how feasible it is to map database behavior across impls. it's worth giving it a try, I think!